src/ecmult_gen_prec.h

```diff
@@ -0,0 +1,16 @@
+/***********************************************************************
+ * Copyright (c) 2013, 2014, 2015 Pieter Wuille, Gregory Maxwell       *
+ * Distributed under the MIT software license, see the accompanying    *
+ * file COPYING or https://www.opensource.org/licenses/mit-license.php.*
+ ***********************************************************************/
+
+#ifndef SECP256K1_ECMULT_GEN_PREC_H
+#define SECP256K1_ECMULT_GEN_PREC_H
+
+#include "ecmult_gen.h"
+
+static const size_t ECMULT_GEN_PREC_TABLE_SIZE = ECMULT_GEN_PREC_N(ECMULT_GEN_PREC_BITS) * ECMULT_GEN_PREC_G(ECMULT_GEN_PREC_BITS) * sizeof(secp256k1_ge_storage);
```

Comment:
Nit: I couldn't find any references to this; should it be either (1) removed or (2) be a `#define` with a `bits` parameter so that it can be used in `gen_ecmult_gen_static_prec_table.c`?